pco_cli/src/input/audio.rs

```diff
@@ -11,12 +11,26 @@ pub fn get_wav_field(path: &Path) -> Result<Option<Field>> {
   // this is excessively slow, but easy for now
   let mut file = File::open(path)?;
   let (header, _) = wav::read(&mut file)?;
-  let dtype = match header.bytes_per_sample {
-    1 | 2 => Ok(DataType::Int16),
-    3 => Ok(DataType::Int32),
-    4 => Ok(DataType::Float32),
+  let dtype = match header.audio_format {
```

Comment:
Can we match both audio_format and bits_per_sample at once instead of nesting?

Reply:
Definitely! Thought it was necessary to get accurate error messages at first, but this is ofc much nicer.